src/utils/__mocks__/db.js

```diff
@@ -0,0 +1,109 @@
+export const mockData = {
+  index_0: 'data_0',
+  index_1: 'data_1',
+  index_2: 'data_2',
+  index_3: 'data_3',
+  index_4: 'data_4',
+};
+
+export const mockDataFirstItem = {
+  index_0: 'data_0_first_item',
+  index_1: 'data_1_first_item',
+  index_2: 'data_2_first_item',
+  index_3: 'data_3_first_item',
+  index_4: 'data_4_first_item',
+};
+
+export const mockDataLastItem = {
+  index_0: 'data_0_last_item',
+  index_1: 'data_1_last_item',
+  index_2: 'data_2_last_item',
+  index_3: 'data_3_last_item',
+  index_4: 'data_4_last_item',
+};
+
+export default {
+  // eslint-disable-next-line no-unused-vars
+  select: jest.fn().mockImplementation((sql, sqlParams) => {
+    if (sql.startsWith('SELECT * FROM tests LIMIT 6 OFFSET 10')) {
+      return Promise.resolve([
+        { ...mockDataFirstItem },
+        { ...mockData },
+        { ...mockData },
+        { ...mockData },
+        { ...mockDataLastItem },
+      ]);
+    }
+
+    if (
+      sql.startsWith('SELECT * FROM tests LIMIT 6') ||
+      sql.startsWith('SELECT * FROM total_tests LIMIT 6')
+    ) {
+      return Promise.resolve([
+        { ...mockDataFirstItem },
+        { ...mockData },
+        { ...mockData },
+        { ...mockData },
+        { ...mockDataLastItem },
+        { ...mockData },
+      ]);
+    }
+
+    if (sql.startsWith('SELECT * FROM total_tests')) {
```

Comment:
Ohh, if we're mocking a data on this way, I think in the long run we can just use sqlite and populate it on runtime, rather than checking if the statement `startsWith`

Reply:
Not intending to run actual db queries on unit tests for now. Everything is being mocked